src/zarr/api/asynchronous.py

```diff
@@ -666,6 +662,54 @@ async def group(
         )
 
 
+async def create_group(
+    *,
+    store: StoreLike,
+    path: str | None = None,
+    overwrite: bool = False,
+    zarr_format: ZarrFormat | None = None,
+    attributes: dict[str, Any] | None = None,
+    storage_options: dict[str, Any] | None = None,
+) -> AsyncGroup:
+    """Create a group.
+
+    Parameters
+    ----------
+    store : Store or str
+        Store or path to directory in file system.
+    path : str, optional
+        Group path within store.
+    overwrite : bool, optional
+        If True, pre-existing data at ``path`` will be deleted before
+        creating the group.
+    zarr_format : {2, 3, None}, optional
+        The zarr format to use when saving.
```

Comment:
```suggestion
        The zarr format to use when saving. If ``None`` (the default), the zarr format will be retrieved from the current state of the zarr config object.
```

Reply:
or something like this. there should be a clear path for people to follow to figure out what the default version will be.